core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2193,9 +2193,11 @@ private[spark] class DAGScheduler(
    * Return true when:
    *  1. Waiting for decommission start
    *  2. Under decommission process
-   * Return false when:
-   *  1. Stopped or terminated after finishing decommission
-   *  2. Under decommission process, then removed by driver with other reasons
+   *  3. Stopped or terminated after finishing decommission
+   *  4. Under decommission process, then removed by driver with other reasons
```

Comment:
This seems to change the original behavior. Would it affect any existing callers?

Reply:
There is only one caller for this, right ?
We should make it `private` instead of `private[scheduler]` to remove the confusion.

Reply:
Yes, upated.